ross/rotor_assembly.py

```diff
@@ -3524,8 +3524,8 @@ def rotor_example_6dof():
     >>> camp6.plot(plot_type="matplotlib")
     >>> plt.show()
     >>> modal6 = rotor6.run_modal(speed=0)
-    >>> np.round(modal6.wd[:6])
-    array([  0.,   0.,  48.,  92.,  96., 275.])
+    >>> modal6.wd[:6]
+    array([  0.        ,  47.62157215,  91.79835717,  96.29386819,  274.51274397, 296.49903736])
```

Comment:
Hello @verg1lio!

I was checking for errors reported by APPVEYOR and this example has something wrong. You've changed the output of `modal6.wd[:6]` and now we're getting an error. It seems that the output you set before was correct, look:

![image](https://user-images.githubusercontent.com/45969994/80371089-b3562e00-8867-11ea-83ec-e57da59d4bc6.png)

Reply:
Hello @rodrigomoliveira1 , good morning! We had the result with 2 rigid body modes when utilizing the axial stiffness of the bearing equal zero. But when we applied a reasonable value it does change to having only one rigid body mode. I suppose that we are getting still 2 rigid body modes because of an error in the entry data for the example. I will be looking into it now, and will try to fix the issue.

Reply:
I re-ran the same example manually, in a python terminal here, and got the expected results without errors. Also tested the test function examples, getting a pass from the test explorer. Maybe the APPVEYOR is using the null value for the axial bearing stiffness?

Reply:
I don't know. 

Try checking if the code you just run is the same you uploaded to this PR.
Or, try downloading your own branch from github and running it on another instance. It have already happened to me, getting different results from the local code and the repository code. I changed something and forgot to commit it.

Reply:
I agree with @rodrigomoliveira1 , maybe you have something different in your local code @verg1lio .
I have created a branch from your PR and I am getting two errors in the `rotor_assembly.py` docstrings.
I have also tried to check Travis CI, but it looks like we are getting errors from isort. I have created an issue #538 to discuss if we should remove isort from our CI.

Reply:
Alright, we will look into this to see where is the problem coming from.

Reply:
Ok... we detected the problem. 

When executing the tests individually, it passes. When running all the tests from "test_rotor_assembly" together, it fails with your error message. 

Have you ever encountered this issue before?

Reply:
Check if you have duplicated ROSS folder salved on another diretory.

When you run it individually, the code refers to the files you just opened and you're using right now. When running all the test at once, maybe, it refers to another ROSS folder.

And check if, in the test files, the imports have `ross.`

Reply:
We are seeing the same problem on different machines. So, we will check the first 6 non-zero terms of the natural frequencies, to avoid any problem issuing from the "eig" solution on different machines. It should solve the problem. 

I will get back to you here once that is solved.

Reply:
We noticed the same problem related to the "eig" solution on different machines and different code runs, due to the "eig" coding. There are a few comments like the following:

@pytest.mark.skip(reason="Different evector order when not sorted")

and the problem we are facing is most likely the same.